### DISCUSSION TITLE
[Gamebreaking Bug] Boss Rage - ff2r version 1.1.184

### DISCUSSION THREAD

**Discussion by Psychicpie** [Batfoxkid/Freak-Fortress-2-Rewrite#186] [General]
In the newest update of ff2r all the bosses rages are bugged. You cannot build rage at all as it remains at 0%. This happens both with a fresh instal and when updating from the previous version. 

Note: When setting ragedamage to 1 you do get 100% rage and it works. Which leads me to believe that it requires the entire ragedamage value to be met with a single hit. This would mean that it isn't adding up the damage.

**Comment by naydef** [comment]
Yep, that is actually the case, however you should have made an issue report instead for better visibility.

**Reply by naydef** [reply]
This is now fixed.
